docs/ref/models/expressions.txt

```diff
@@ -252,6 +252,16 @@ The ``Func`` API is as follows:
         A class attribute that denotes the character used to join the list of
         ``expressions`` together. Defaults to ``', '``.
 
+    .. attribute:: arity
```

Comment:
I don't know if "arity" is the simplest term. I had to look up what it meant.. Something like "num_expressions" might be more intuitive. @jarshwah any opinion?